draft-ietf-cats-framework.md

```diff
@@ -381,13 +380,63 @@ Note that, depending on the design considerations and service requirements, per-
+
+* **Centralized model**:
+: Computing metrics are collected by a centralized control plane, and then the centralized control plane computes the forwarding path for service requests and syncs up with the Ingress CATS-Forwarder. In this model, C-PS is implemented in the centralized control plane.
+
+* **Hybrid model**:
+: Is a combination of distribution and centralized models.
+: A part of computing metrics are distributed among involved network devices, and others may be collected by a centralized control plane. For example, some static information (e.g., capabilities information) can be distributed among network devices since they are quite stable (change infrequently). Frequent changing information (e.g., resource utilization) can be collected by a centralized control plane to avoid frequent flooding in the distributed control plane. Service scheduling function can be performed by a centralized control plane and/or the CATS-Forwarder. The entire or partial C-PS function may be implemented in the centralized control plane, depending on the specific implementation and deployment.
+
+## Verify Correct Operations
+
+CATS may be implemented by extending some existing control plane protocols, such as BGP or PCEP. A CATS implementation must log error events for better network management and operation. Means to assess the reachability and trace CATS paths should be supported.
+
+## Impact on Network Operations
+
+Computing metrics are collected and distributed in CATS. A new function is needed to be deployed to manage the cooperation between network elements and computing elements. For example, this function may be provided by an orchestrator connecting with C-SMA and C-NMA. This might bring more complexity of the network management, especially if this function is not leveraged for other purposes beyond CATS.
+
 # CATS Framework Workflow {#sec-cats-workflow}
 
 The following subsections provide an overview of how the CATS workflow operates.
 
 ## Service Announcement
 
-A service is associated with a unique identifier called a CS-ID. A CS-ID may be a network identifier, such as an IP address. The mapping of CS-IDs to network identifiers may be learned through a name resolution service (e.g., DNS {{?RFC1034}}). Note that CATS framework does not assume nor preclude any specific name resolution service.
+In order to provide service to clients, a service needs to be known by the clients before the clients requesting for service. Typically, in the current practices of the web services, a service provider will register the IP addresses of the servers in a DNS server. A client can request for server IP addresses by the URL, and it can receive the IP addresses from the DNS server.
```

Comment:
I would delete the first sentence as this is over-simplifying required DNS configurations. There are several RR types out there. More importantly, I suggest we focus on CATS specifics.

Reply:
It is true that this is over-simplifying the DNS configuration. The reason is to focus on CATS specific part. OK, I will delete the whole text in line 437.